db.json

```diff
@@ -6927,6 +6927,10 @@
     "source": "apache",
     "extensions": ["xwd"]
   },
+  "image/vnd.ms-dds" : {
```

Comment:
Please do not make a PR against this file, per instructions at https://github.com/jshttp/mime-db#contributing

If you do want to include it, it should be the output from `npm run build` (this looks like it was done by hand instead).

Reply:
Ok yes I see, I looked at somone elses PR as an example I will remove

Reply:
@dougwilson thanks for all the help, I'm more familiar now with the project, so any future PRs will be better

Reply:
No problem at all.